test/modules/parser/parsers/parse-xviz-stream.spec.js

```diff
@@ -153,6 +162,10 @@ tape('parseStreamVariable#simple v2', t => {
   const result = parseStreamVariable(testData, '/test', time);
   t.deepEquals(result, expected, 'variables parsed properly');
 
+  setXVIZConfig({DYNAMIC_STREAM_METADATA: true});
+  const {__metadata} = parseStreamVariable(testData, '/test', time);
+  t.is(__metadata.category, 'VARIABLE', 'metadata generated');
+
```

Comment:
Should `scalar_type` be verified here?

Reply:
How does `scalar_type` in metadata work when there are multiple types in this stream?

Reply:
Ideally there shouldn't be.

The intention was that the multiple elements only really make sense if you assign an `object_id` so you can associate the values with something meaningful (if they share the stream id like `/object/speed_prediction`) 
Otherwise I don't know how you interpret or use the elements, like if I have `/object/velocity_prediction` with multiple entries but w/o an object_id for each one, I don't know how to make sense of the data then.

The same issue fundamentally happens with `primitives` in that we assume each primitive is of a single `primitive_type`.  Like you can't have `/object/ufo` as both circles and polygons.   It doesn't look like the Schema enforces this restriction, but i'm pretty sure our code won't really handle them right.

Reply:
I am saying, either the relationship is implicit, and the `variable` would have only 1 entry, or it must be explicit with an `object_id` and multiple entries.  Otherwise it is hard to see how the data is useful.

Reply:
I'd be fine with taking the type of the first entry.  It is likely what we do elsewhere if necessary.

Reply:
Looks like `getVariableData()` returns the first entry if finds too